.github/workflows/nightly-tests.yaml

```diff
@@ -4,24 +4,17 @@ env:
   GITHUB_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }} # necessary to pass upgrade tests
 
 on:
-  # https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07
-  # Minute [0,59]
-  # Hour [0,23]
-  # Day of the month [1,31]
-  # Month of the year [1,12]
-  # Day of the week ([0,6] with 0=Sunday)
-  schedule:
-    - cron: "0 5 * * *" # every day @ 05:00 UTC, run tests against latest main
-    - cron: "0 6 * * 1" # monday    @ 06:00 UTC, run expanded tests against v1.17.x
-    - cron: "0 7 * * 1" # monday    @ 07:00 UTC, run expanded tests against v1.16.x
-    - cron: "0 8 * * 1" # monday    @ 08:00 UTC, run expanded tests against v1.15.x
+  # Since this is an LTS branch, we removed the schedule, since that is defined on the `main` branch
+  # schedule:
+  # -
   workflow_dispatch:
     inputs:
       branch:
         description: "The branch to run tests against"
         type: choice
         options:
           - main
+          - v1.18.x
```

Comment:
If we are going to remove the jobs that run for all of the options (everywhere `inputs.branch ==` got deleted) except for `v1.18.x`, should we make it the only option?